internal/network/data_sender.go

```diff
@@ -29,6 +29,7 @@ import (
 const fingerprintTopic = "fingerprint"
 const canDumpTopic = "protocol/canbus/dump"
 const deviceStatusTopic = "status"
+const deviceNetworkTopic = "network"
```

Comment:
just heads up we'll need to add a new mqtt topic in cluster-helm-charts

Reply:
Should we create it now in cluster-helm-charts and drop all the messages for the network topic? OR not create it at all, since if you publish to a topic that does not exist, the broker will automatically generate that topic, and I assume the message will be discarded by broker since there are no consumers of it.

Reply:
I believe Kafka will still store the messages even if there is no consumer since it supports replaying and has it's own persistent storage.